src/routes/project.js

```diff
@@ -54,6 +54,16 @@ router.post('/:id/questions', loginRequired, checkIdParams, projectPermission, a
   ctx.body = await Project.addQuestion(ctx.params.id, ctx.request.body)
 })
 
+router.post('/:id/questions/batch', loginRequired, checkIdParams, projectPermission, async (ctx) => {
```

Comment:
would you please write test for this api thanks

Reply:
@jeyem 

I added a test for it but Removing Questions keeps getting failed, i am guessing that the routes have conflict issues.
```
● remove questions

    expect(received).toBe(expected) // Object.is equality

    Expected: 200
    Received: 400

      196 |         .send(question)
      197 |
    > 198 |       expect(response.status).toBe(200)
          |                               ^
      199 |     }
      200 |
      201 |     const response = await request

      at toBe (test/e2e/globals/project.js:198:31)
          at runMicrotasks (<anonymous>)
```